addons/website_sale/tests/test_website_sale_access.py

```diff
@@ -0,0 +1,33 @@
+from odoo.fields import Command
+from odoo.tests.common import HttpCase, tagged
+
+
+@tagged('post_install', '-at_install')
+class TestWebsiteSaleAccess(HttpCase):
+
+    @classmethod
+    def setUpClass(self):
+        super().setUpClass()
+        self.website = self.env['website'].get_current_website()
+        self.env['ir.ui.view'].search([
+            ('key', '=', 'website_sale.products_categories')
+        ]).active = True
+        self.env['ir.ui.view'].search([
+            ('key', '=', 'website_sale.option_collapse_products_categories')
+        ]).active = False
```

Comment:
Also from `BaseCommon` class.
```suggestion
        cls.quick_ref('website_sale.products_categories').active = True
        cls.quick_ref('website_sale.option_collapse_products_categories').active = True
```